docs/user-guide/nemoaligner.rsts

```diff
@@ -1,8 +1,8 @@
 
-.. _model-aligner-intro:
+.. _nemo-aligner-intro:
 
-Model Alignment
-!!!!!!!!!!!!!!!
+NeMo-Aligner
```

Comment:
suggest title change to be consistent with other titles [Model Alignment by Rejection Sampling, Model Alignment by Supervised Fine-Tuning (SFT), Model Alignment by SteerLM Method]

Model Alignment by NeMo-Aligner or Model Alignment using the NeMo-Aligner Toolkit